### PR TITLE
[zh-cn]: update the translation of `CanvasRenderingContext2D.createPattern()` method

### PR DESCRIPTION
**Pull request by tianyeeT**
…ttern()` method

### CODE REVIEW

**Review by yin1999 — COMMENTED** [review]
标题

File: files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md [hunk 1]
```diff
@@ -1,53 +1,63 @@
 ---
 title: CanvasRenderingContext2D.createPattern()
```
Comment (tianyeeT) [inline files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md:2]:
```suggestion
title: CanvasRenderingContext2D：createPattern() 方法
```

File: files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md [hunk 2]
```diff
@@ -1,53 +1,63 @@
...
+l10n:
+  sourceCommit: 1f216a70d94c3901c5767e6108a29daa48edc070
 ---
 
 {{APIRef}}
 
-**`CanvasRenderingContext2D.createPattern()`** 是 Canvas 2D API 使用指定的图像 ({{domxref("CanvasImageSource")}}) 创建模式的方法。它通过 repetition 参数在指定的方向上重复元图像。此方法返回一个{{domxref("CanvasPattern")}}对象。
+Canvas 2D API 的 **`CanvasRenderingContext2D.createPattern()`** 方法用于使用指定的图像或重复创建模式的方法。此方法返回一个 {{domxref("CanvasPattern")}} 对象。
+
+这个方法并不直接在画布上绘制任何内容。它所创建的图案必须赋值给 属性 {{domxref("CanvasRenderingContext2D.fillStyle")}} 或 {{domxref("CanvasRenderingContext2D.strokeStyle")}}，之后才会应用于接下来的绘制操作。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md:12]:
```suggestion
这个方法并不直接在画布上绘制任何内容。它所创建的图案必须赋值给 {{domxref("CanvasRenderingContext2D.fillStyle")}} 或 {{domxref("CanvasRenderingContext2D.strokeStyle")}}，之后才会应用于接下来的绘制操作。
```

File: files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md [hunk 3]
```diff
@@ -1,53 +1,63 @@
...
-    - {{domxref("HTMLVideoElement")}} ({{HTMLElement("video")}}),
-    - {{domxref("HTMLCanvasElement")}} ({{HTMLElement("canvas")}}),
-    - {{domxref("CanvasRenderingContext2D")}},
-    - {{domxref("ImageBitmap")}},
-    - {{domxref("ImageData")}},
-    - {{domxref("Blob")}}.
+    - {{domxref("HTMLImageElement")}} ({{HTMLElement("img")}})
+    - {{domxref("SVGImageElement")}} ({{SVGElement("image")}})
+    - {{domxref("HTMLVideoElement")}} ({{HTMLElement("video")}}，通过使用捕获视频)
+    - {{domxref("HTMLCanvasElement")}} ({{HTMLElement("canvas")}})
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md:29]:
```suggestion
    - {{domxref("HTMLImageElement")}}（{{HTMLElement("img")}}）
    - {{domxref("SVGImageElement")}}（{{SVGElement("image")}}）
    - {{domxref("HTMLVideoElement")}}（{{HTMLElement("video")}}，通过使用捕获视频）
    - {{domxref("HTMLCanvasElement")}}（{{HTMLElement("canvas")}}）
```

File: files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md [hunk 4]
```diff
@@ -1,53 +1,63 @@
...
-    - `"repeat-x"` (horizontal only),
-    - `"repeat-y"` (vertical only),
-    - `"no-repeat"` (neither).
+    - `"repeat"`（两个方向都重复）
+    - `"repeat-x"`（仅水平方向重复）
+    - `"repeat-y"`（仅垂直方向重复）
+    - `"no-repeat"`（两个方向都不重复）
 
-    如果为空字符串 (`''`) 或 {{jsxref("null")}} (但不是 {{jsxref("undefined")}})，repetition 将被当作"repeat"。
+    如果 `repetition` 被指定为空字符串 (`""`) 或 [`null`](/zh-CN/docs/Web/JavaScript/Reference/Operators/null)（但不是 {{jsxref("undefined")}}），则将使用 `"repeat"` 的值。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md:43]:
```suggestion
    如果 `repetition` 被指定为空字符串（`""`）或 [`null`](/zh-CN/docs/Web/JavaScript/Reference/Operators/null)（但不是 {{jsxref("undefined")}}），则将使用 `"repeat"` 的值。
```

File: files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md [hunk 5]
```diff
@@ -1,53 +1,63 @@
...
+title: CanvasRenderingContext2D：createPattern() 方法
 slug: Web/API/CanvasRenderingContext2D/createPattern
+l10n:
+  sourceCommit: 1f216a70d94c3901c5767e6108a29daa48edc070
 ---
 
 {{APIRef}}
 
-**`CanvasRenderingContext2D.createPattern()`** 是 Canvas 2D API 使用指定的图像 ({{domxref("CanvasImageSource")}}) 创建模式的方法。它通过 repetition 参数在指定的方向上重复元图像。此方法返回一个{{domxref("CanvasPattern")}}对象。
+Canvas 2D API 的 **`CanvasRenderingContext2D.createPattern()`** 方法用于使用指定的图像或重复创建模式的方法。此方法返回一个 {{domxref("CanvasPattern")}} 对象。
```
Comment (yin1999) [inline files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md:10]:
```suggestion
Canvas 2D API 的 **`CanvasRenderingContext2D.createPattern()`** 方法用于使用指定的图像或重复创建图案。此方法返回一个 {{domxref("CanvasPattern")}} 对象。
```

File: files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md [hunk 6]
```diff
@@ -1,53 +1,63 @@
...
-    - {{domxref("HTMLImageElement")}} ({{HTMLElement("img")}}),
-    - {{domxref("HTMLVideoElement")}} ({{HTMLElement("video")}}),
-    - {{domxref("HTMLCanvasElement")}} ({{HTMLElement("canvas")}}),
-    - {{domxref("CanvasRenderingContext2D")}},
-    - {{domxref("ImageBitmap")}},
-    - {{domxref("ImageData")}},
-    - {{domxref("Blob")}}.
+    - {{domxref("HTMLImageElement")}}（{{HTMLElement("img")}}）
+    - {{domxref("SVGImageElement")}}（{{SVGElement("image")}}）
+    - {{domxref("HTMLVideoElement")}}（{{HTMLElement("video")}}，通过使用捕获视频）
```
Comment (yin1999) [inline files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md:28]:
```suggestion
    - {{domxref("HTMLVideoElement")}}（{{HTMLElement("video")}}，通过使用视频捕获）
```

File: files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md [hunk 7]
```diff
@@ -1,53 +1,63 @@
...
-    - `"repeat-x"` (horizontal only),
-    - `"repeat-y"` (vertical only),
-    - `"no-repeat"` (neither).
+    - `"repeat"`（两个方向都重复）
+    - `"repeat-x"`（仅水平方向重复）
+    - `"repeat-y"`（仅垂直方向重复）
+    - `"no-repeat"`（两个方向都不重复）
 
-    如果为空字符串 (`''`) 或 {{jsxref("null")}} (但不是 {{jsxref("undefined")}})，repetition 将被当作"repeat"。
+    如果 `repetition` 被指定为空字符串（`""`）或 [`null`](/zh-CN/docs/Web/JavaScript/Reference/Operators/null)（但不是 {{jsxref("undefined")}}），则将使用 `"repeat"` 的值。
```
Comment (yin1999) [inline files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md:43]:
```suggestion
    如果 `repetition` 被指定为空字符串（`""`）或 [`null`](/zh-CN/docs/Web/JavaScript/Reference/Operators/null)（但不是 {{jsxref("undefined")}}），则将使用 `"repeat"` 的值。
    [`null`](/zh-CN/docs/Web/JavaScript/Reference/Operators/null) 值与空字符串（`""`）的处理方式相同：两者都是 `"repeat"` 的同义词。
```

File: files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md [hunk 8]
```diff
@@ -1,53 +1,63 @@
...
 
 ### 返回值
 
 - {{domxref("CanvasPattern")}}
   - : 描述模式的不透明对象
```
Comment (yin1999) [inline files/zh-cn/web/api/canvasrenderingcontext2d/createpattern/index.md:48]:
```suggestion
  - : 描述图案的不透明对象
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**